### ISSUE TITLE
Setting Multiplayer Defaults

### ISSUE THREAD

**Issue by Aevrin** [wesnoth/wesnoth#6678]
### Describe the desired feature

Would it be possible to have a preferences page for Multiplayer defaults (e.g. setting starting gold value, setting other players to default to AI's, setting default income, or just have it keep the last setting the way the map chooser does.

**Comment by Wedge009** [comment]
This sounds like a super-set of #1556 and somewhat related to #4440.

**Comment by Aevrin** [comment]
Somewhat related to #4440, but different in that these are already exposed values, there isn't a way to save the settings, but the same at #1556.

**Comment by Wedge009** [comment]
Yes, sorry, was more a remark for the team than for you specifically.

**Comment by Vultraz** [comment]
#1556 fixed in 8a960fca68872dab1c0aaa812bbc581eeb8d81f4.
